runtime/near-vm-logic/src/dependencies.rs

```diff
@@ -138,7 +139,7 @@ impl std::fmt::Display for ExternalError {
             InvalidAccountId => write!(f, "VM Logic returned an invalid account id"),
             InvalidMethodName => write!(f, "VM Logic returned an invalid method name"),
             InvalidPublicKey => write!(f, "VM Logic provided an invalid public key"),
-            StorageError => write!(f, "Storage error"),
+            StorageError(e) => write!(f, "Storage error: {:?}", e),
```

Comment:
I wonder if it makes sense to print `e` if it is a serialized blob of data.

Reply:
printing is still TODO